packages/salesforcedx-vscode-core/src/commands/forceCreateManifest.ts

```diff
@@ -39,6 +40,12 @@ export class ManifestCreateExecutor extends LibraryCommandletExecutor<string> {
                    token?: vscode.CancellationToken): Promise<boolean> {
     if (this.sourcePaths) {
       const componentSet = ComponentSet.fromSource(this.sourcePaths);
+      if (componentSet.apiVersion) {
```

Comment:
Checks if there is an existing apiVersion before fetching the sourceApiVersion. Tests that use a dummy file will fail otherwise.